gateway/src/apicast/proxy.lua

```diff
@@ -126,6 +127,31 @@ local function output_debug_headers(service, usage, credentials)
   end
 end
 
+-- Converts a usage to the format expected by the 3scale backend client.
+local function parse_usage(usage)
```

Comment:
Just a nitpick. I think this should be called `serialize_usage` or `format_usage`. It is not parsing, but rather transforming, from one format to another.

Reply:
:+1: